src/reports/ChartReportPage.jsx

```diff
@@ -161,7 +164,14 @@ const ChartReportPage = () => {
                 formatter={(value, key) => [value, positionAttributes[key]?.name || key]}
                 labelFormatter={(value) => formatTime(value, 'seconds')}
               />
-              <Line type="monotone" dataKey={type} stroke={theme.palette.primary.main} />
+              <Brush
+                dataKey={timeType}
+                height={30}
+                stroke={theme.palette.primary.main}
+                tickFormatter={() => ''}
+                onChange={(range) => handleBrushChange(range)}
+              />
+              <Line type="monotone" dataKey={type} stroke={theme.palette.primary.main} dot={false} activeDot={{ r: 6 }} />
```

Comment:
Can we move attributes on separate lines for better readability please.

Reply:
This has been done. Thank you